cassandra-analytics-release/finish_release.sh

```diff
@@ -0,0 +1,210 @@
+
+#
+# Rename the git tag, removing the -tentative suffix
+#
+
+execute "cd $cassandra_dir"
+
+echo "Tagging release ..." 1>&3 2>&4
+execute "git checkout $release-tentative"
+
+# Ugly but somehow 'execute "git tag -a cassandra-analytics-$release -m 'Apache Cassandra $release release' "' doesn't work
+echo "Apache Cassandra Analytics $release release" > "_tmp_msg_"
+execute "git tag -a cassandra-analytics-$release -F _tmp_msg_"
+rm _tmp_msg_
+execute "git push $git_asf_remote refs/tags/cassandra-analytics-$release"
+execute "git tag -d $release-tentative"
+execute "git push $git_asf_remote :refs/tags/$release-tentative"
+
+#
+# Move staging artifacts to release distribution location
+#
+
+tmp_dir=`mktemp -d`
+cd $tmp_dir
+echo "Apache Cassandra Analytics $release release" > "_tmp_msg_"
+execute "svn mv -F _tmp_msg_ https://dist.apache.org/repos/dist/dev/cassandra/cassandra-analytics/${release} https://dist.apache.org/repos/dist/release/cassandra/cassandra-analytics/"
+rm _tmp_msg_
+
+# Cleaning up
+execute "cd $cassandra_dir"
+rm -rf $tmp_dir
+
+# Restore stdout/stderr (and close temporary descriptors) if not verbose
+[ $verbose -eq 1 ] || exec 1>&3 3>&- 2>&4 4>&-
+
+mail_file="$mail_dir/mail_release_$release"
+[ ! -e "$mail_file" ] || rm $mail_file
+
+echo "[RELEASE] Apache Cassandra Analytics $release released" > $mail_file
+echo "" >> $mail_file
+echo "The Cassandra team is pleased to announce the release of Apache Analytics Cassandra version $release." >> $mail_file
+echo "" >> $mail_file
+echo "" >> $mail_file
+echo "Downloads of source and binary distributions are available here:" >> $mail_file
+echo "" >> $mail_file
+echo " https://dlcdn.apache.org/cassandra/cassandra-analytics/$release/" >> $mail_file
+echo "" >> $mail_file
+echo "" >> $mail_file
+echo "The Maven artifacts can be found at: " >> $mail_file
+echo "" >> $mail_file
+echo " https://repo.maven.apache.org/maven2/org/apache/cassandra/" >> $mail_file
+echo "" >> $mail_file
+echo "These will be mirrored to other repositories." >> $mail_file
+echo "" >> $mail_file
+echo "" >> $mail_file
+echo "As always, please review the changes[1] and pay attention to the release notes[2]. Let us know[3] if you were to encounter any problem." >> $mail_file
+echo "" >> $mail_file
+echo "" >> $mail_file
+echo 'Enjoy!' >> $mail_file
+echo "" >> $mail_file
+echo "[1]: CHANGES.txt https://github.com/apache/cassandra-analytics/blob/cassandra-analytics-$release/CHANGES.txt" >> $mail_file
+echo "[2]: NEWS.txt https://github.com/apache/cassandra-analytics/blob/cassandra-analytics-$release/NEWS.txt" >> $mail_file
+echo "[3]: https://issues.apache.org/jira/browse/CASSANALYTICS" >> $mail_file
+
+
+echo 'Done deploying artifacts. Please make sure to:'
+echo ' 1) "Release" the staging repository from repository.apache.org'
+echo ' 2) wait for the artifacts to sync at https://downloads.apache.org/cassandra/'
+echo ' 3) update the website (TODO provide link)'  # TODO - this is old info and needs updating..
```

Comment:
any idea how/where you'll add info about cassandra-analytics to the downloads page yet ? 
https://cassandra.apache.org/_/download.html